torch/_export/non_strict_utils.py

```diff
@@ -259,9 +270,13 @@ def produce_guards_and_solve_constraints(
     )
     dim_constraints.remove_redundant_dynamic_results()
     forced_specializations = dim_constraints.forced_specializations()
-    msg = dim_constraints.prettify_results(
-        original_signature, constraint_violation_error, forced_specializations
-    )
+    if not _is_torch_jit_trace:
+        msg = dim_constraints.prettify_results(
+            original_signature, constraint_violation_error, forced_specializations
+        )
+    else:
+        # FIXME(ycao): This is a hack to get around missing signature from ScriptMethod
+        msg = "dummy constraint violation message"
```

Comment:
Wonder if it's better to just have `msg = ""` here, analogous to other zero-d out things in this mode. It might even be better to have a top-level dict of all the things we're zero-ing out, something like `torch_jit_trace_config = {"msg": "", "original_signature": None, ...}` etc. so that we have a good account of all the FIXMEs.